domains/thehcj.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "thehcj"
+    },
```

Comment:
Please add either a email field or leave the email field blank and add another social.

Reply:
I'll add my Bluesky account now